### DISCUSSION TITLE
How to Merge Multiple Layers into One Clickable Layer in QField?

### DISCUSSION THREAD

**Discussion by SIG-CCBE** [opengisch/QField#5746] [Q&A]
Hello, 
is it possible to merge multiple layers into a single one? 
The idea is to have only one clickable layer displayed in QField instead of several.
Thank you in advance.

**Comment by SeqLaz** [comment]
You can first promote your Multisurface to single part by using the Processing tool called "Multipart to singleparts"

![image](https://github.com/user-attachments/assets/3c730b56-7ace-4310-a0d6-54c012c0e1c4)
